src/server/bin/kibana.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/sh
```

Comment:
We set the perms when we build the package because the copy command doesn't copy the perms correctly. :-1: